src/pybind/mgr/restful/module.py

```diff
@@ -465,6 +475,22 @@ def handle_command(self, inbuf, command):
                 ""
             )
 
+        elif command['prefix'] == "restful disable-http-auth":
+            self.disable_auth = True
```

Comment:
You will need to store these somewhere. Otherwise, they won't persist accross daemon restarts. You can use `set_store`/`get_localized_store` functions for that.